### PR TITLE
Feature/product details

### PR DESCRIPTION
**Pull request by rubiocanino**
Adding the feature of PDP including the button that include the quantity.

### CODE REVIEW

File: theme6/source/javascripts/details.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+function selectimage(image) {
```
Comment (magmahound) [inline theme6/source/javascripts/details.js:1]: 'selectimage' is defined but never used.

File: theme6/source/stylesheets/site.css.scc [hunk 2]
```diff
@@ -0,0 +1,162 @@
+
```
Comment (Ilyeo) [inline theme6/source/stylesheets/site.css.scc]: @rubiocanino check file, it should be `site.css` not `site.css.scc`

**Review by Ilyeo — DISMISSED** [review]
cool looks good to me 👍

**Review by luisarmandodv — APPROVED** [review]
:shipit:

**Pull request merged**